src/MicroEd-Tests/MDFileTest.class.st

```diff
@@ -0,0 +1,97 @@
+Class {
+	#name : 'MDFileTest',
+	#superclass : 'TestCase',
+	#instVars : [
+		'memoryFS',
+		'fileExample',
+		'fileWithoutFileReference',
+		'toDelete',
+		'fileReferenceWithName'
+	],
+	#category : 'MicroEd-Tests',
+	#package : 'MicroEd-Tests'
+}
+
+{ #category : 'running' }
+MDFileTest >> setUp [
+
+	super setUp.
+	memoryFS := FileSystem memory.
+	toDelete := OrderedCollection new.
+	fileWithoutFileReference := MDFile new.
+	fileExample := memoryFS / 'example.txt'.
+	fileReferenceWithName := (memoryFS / 'anotherFile.txt')
+		                         ensureCreateFile.
+	toDelete add: fileExample. 
+	fileExample writeStreamDo: [ :stream |
+		stream nextPutAll: 'example file' ]
+]
+
+{ #category : 'running' }
+MDFileTest >> tearDown [ 
+
+	toDelete
+		select: [ :path | memoryFS exists: path ]
+		thenDo: [ :path | path ensureDelete ].
+	super tearDown
+]
+
+{ #category : 'running' }
+MDFileTest >> testMDFileCreationOverwriteExistingFile [
+
```

Comment:
If you want to keep this test for another development iteration add self skip.
so that we know that the test is not green. 
Having empty green tests is a bad practice.